modules/500-cilium-hubble/images/ui-frontend/patches/hubble-ui-vm-name.patch

```diff
@@ -0,0 +1,86 @@
+diff --git a/src/components/EndpointCardHeader/helpers.ts b/src/components/EndpointCardHeader/helpers.ts
```

Comment:
Also, need a README.md file with patch description.
Like [this one](https://github.com/deckhouse/deckhouse/blob/main/modules/021-cni-cilium/images/bin-cilium/patches/README.md).

Reply:
README.md is added.